diff --git a/node_modules/hexo-toc/lib/filter.js b/node_modules/hexo-toc/lib/filter.js
index 5e6905c..6d1ab98 100644
--- a/node_modules/hexo-toc/lib/filter.js
+++ b/node_modules/hexo-toc/lib/filter.js
@@ -25,10 +25,10 @@ exports.heading = function (data) {
     var $title = $(this);
     var title = $title.text();
     var id = toc.slugify(title, options);
-    // $title.attr('id', id);
-    $title.children('a').remove();
-    $title.html( '<span id="' + id + '">' + $title.html() + '</span>' );
-    $title.removeAttr('id');
+    $title.attr('id', id);
+// $title.children('a').remove();
+// $title.html( '<span id="' + id + '">' + $title.html() + '</span>' );
+// $title.removeAttr('id');
 
 
     if (options.anchor) {
